ui/nuxeo-aggregation/nuxeo-checkbox-aggregation.js

```diff
@@ -257,6 +255,19 @@ import { AggregationBehavior } from './nuxeo-aggregation-behavior.js';
       this.setAttribute('tabindex', 0);
     }
 
+    _formatDocCount(docCount) {
+      // Fetch the property value from web-ui-properties.xml
+      const isNumberFormattingEnabled =
+        Nuxeo && Nuxeo.UI && Nuxeo.UI.config && Nuxeo.UI.config.numberFormattingEnabled !== undefined
```

Comment:
no need to check for "!== undefined". remove this part

Reply:
removed